regression/strings-smoke-tests/java_if/test.desc

```diff
@@ -0,0 +1,9 @@
+CORE
+test.class
+--refine-strings --string-max-length 100
+^EXIT=10$
+^SIGNAL=0$
+^\[.*assertion.1\].* line 12.* SUCCESS$
+^\[.*assertion.2\].* line 13.* FAILURE$
+--
+$ignoring
```

Comment:
Should the `$ignoring` expression be a bit more specific to avoid catching some other (future/unrelated) `ignoring`s in the output?